public/apps/login/test/login-page.test.tsx

```diff
@@ -85,6 +86,53 @@ describe('test extractNextUrlFromWindowLocation', () => {
   });
 });
 
+describe('test redirect', () => {
+  test('extract redirect excludes security_tenant when no tenant in local storage', () => {
+    // Trick to mock window.location
+    const originalLocation = window.location;
+    delete window.location;
+    window.location = new URL(
+      "http://localhost:5601/app/login?nextUrl=%2Fapp%2Fdashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d?_g=(filters:!(),refreshInterval:(pause:!f,value:900000),time:(from:now-24h,to:now))&_a=(description:'Analyze%20mock%20flight%20data%20for%20OpenSearch-Air,%20Logstash%20Airways,%20OpenSearch%20Dashboards%20Airlines%20and%20BeatsWest',filters:!(),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!t,title:'%5BFlights%5D%20Global%20Flight%20Dashboard',viewMode:view)"
```

Comment:
these urls can be shortened for clarity: `http://localhost:5601/app/login?nextUrl=%2Fapp%2Fdashboards#/`. Thoughts?

Reply:
Shortened the URLs, but kept one test with the has to verify that the hash is working correctly.